internal/graph/cached_resolver_test.go

```diff
@@ -654,7 +654,7 @@ func TestCachedCheckDatastoreQueryCount(t *testing.T) {
 	ttuLocalChecker.Close()
 
 	require.NoError(t, err)
-	require.Equal(t, uint32(2), res.GetResolutionMetadata().DatastoreQueryCount)
```

Comment:
I'm not sure why in this PR specifically, but I'm able to corroborate that datastore only gets read once with this test.